Automattic/jetpack
# Add JetpackBanner to Scan card in At-a-Glance

Since we're removing the "scary" upgrade buttons, we're using the banner as a more gentle nudge towards upgrading. This PR is currently a **Work In Progress**, as this certainly isn't the best way to accomplish this task. 

To Do:

- [x] Ideally the JetpackBanner should be below the `.dops-section-header` instead of inside the `.jp-dash-item__card` so we don't have to use CSS to change the padding, etc.
- [x] It needs a sensible Tracks event name
- [x] Possibly update the Upgrade HREF (unless we plan on redirecting all traffic on the URI to the new landing page) (see: https://github.com/Automattic/jetpack/pull/12837)

Fixes #12564 

#### Changes proposed in this Pull Request:

* Updates the Security Scanning card in the At-a-Glance view to use the JetpackBanner component to nudge towards an upgrade

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* This is part of the At a Glance Tuneup project

#### Testing instructions:
* Go to At a Glance after connecting Jetpack on a Free plan
* See the JetpackBanner prompting an upgrade in the Security Scanning card

#### Proposed changelog entry for your changes:

* Changes to upgrade messaging in the Security Scanning card

#### Screenshots:

Before:
![58902607-a6b17000-86c0-11e9-8abe-59a8d147cf9b](https://user-images.githubusercontent.com/4297811/59069204-30447780-8852-11e9-8b5b-0879a26a6e7d.png)
 
After:
<img width="530" alt="Security Scanning v3" src="https://user-images.githubusercontent.com/4297811/59224658-a30f6480-8b6a-11e9-9a2e-192c4142ee06.png">
